src/main/java/org/dcsa/bkg/service/impl/BookingServiceImpl.java

```diff
@@ -703,6 +702,7 @@ private BookingTO bookingToDTOWithNullLocations(Booking booking) {
             b -> {
               // update booking with new booking request
               Booking booking = bookingMapper.dtoToBooking(bookingRequest);
+              booking.setUpdatedDateTime(updatedDateTime);
```

Comment:
I did the same fix in a PR before this that was merged, it's weird that it does not conflict.

Reply:
Yeah, I looked into this and can see your point. Not sure why this happened, but the code is not duplicated

Reply:
@Abdiiir make sure you verify that the code isn't overwritten on merge.

Reply:
This should be resolved now and the duplicate line caused from re-basing is removed. 

@douradofrazer waiting for a second review.